docs/topics/browsable-api.md

```diff
@@ -20,9 +20,11 @@ By default, the API will return the format specified by the headers, which in th
 To quickly add authentication to the browesable api, add a routes named `"login"` and `"logout"` under the namespace `"rest_framework"`. DRF provides default routes for this which you can add to your urlconf:
 
 ```python
+from django.urls import include, re_path
+
 urlpatterns = [
     # ...
-    url(r"^api-auth/", include("rest_framework.urls", namespace="rest_framework"))
+    re_path(r"^api-auth/", include("rest_framework.urls", namespace="rest_framework"))
```

Comment:
This don't need be a regex, `path("api-auth/")...` is just fine.

Reply:
Little note about why i suggested re_path and not path (searching stackoverflow):
**_re_path()_** is similar to **_url()_** (both have regex support).
**_path()_** doesn't. ([See Django docs](https://docs.djangoproject.com/en/5.1/topics/http/urls/#using-regular-expressions))
But you are right, most of the docs use **_path()_**